### PR TITLE
IsWallSpell

### PR DESCRIPTION
**Pull request by qndel**

### CODE REVIEW

File: Source/spells.h [hunk 1]
```diff
@@ -16,6 +16,7 @@ enum class SpellCheckResult : uint8_t {
 	Fail_Busy,
 };
 
+bool IsWallSpell(const spell_id spl);
```
Comment (glebm) [inline Source/spells.h:19]: nit: There is no need to specify `const` here. It is passed by value (copied) and what the function does with the copy is its own business.

**Review by glebm — APPROVED** [review]

File: Source/spells.h (hunk 1 above)
Comment (AJenbo) [reply Source/spells.h:19]:
```suggestion
bool IsWallSpell(spell_id spl);
```

File: Source/spells.cpp [hunk 2]
```diff
@@ -108,6 +108,11 @@ void PlacePlayer(int pnum)
 
 } // namespace
 
+bool IsWallSpell(const spell_id spl)
```
Comment (AJenbo) [inline Source/spells.cpp:111]:
```suggestion
bool IsWallSpell(spell_id spl)
```

**Review by AJenbo — APPROVED** [review]

File: Source/spells.h (hunk 1 above)
Comment (qndel) [reply Source/spells.h]: right : D damn I wanted to use const for the first time

**Pull request merged**